### ISSUE TITLE
latexify does not work anymore

### ISSUE THREAD

**Issue by ahaselsteiner** [ahaselsteiner/virocon#66]
It seems that my commit ae0224f169e0bd4394f510461e716271af858d4c ("change virocon motto") has broken the latex representation of the equations. When you show a probabilistic model now, equations are not shown, but ugly text:

![image](https://user-images.githubusercontent.com/11452055/35344508-9d7359b4-012d-11e8-9587-44b7f42ff563.png)

In the commit before that one (548818891ed8318b1298f1ec349d044f62b889ea, "change return_period and state_duration to be floats instead of ints …") it still works.

The strange thing is that in the commit, which broke it, I only changed the readme and the data base (db.sqlite3), see https://github.com/ahaselsteiner/virocon/commit/ae0224f169e0bd4394f510461e716271af858d4c

How can a change in the database break latexify? Or is the problem somewhere else?

**Comment by ahaselsteiner** [comment]
I investigated this further:

I think the error was introduced with the merge fc05268716ad2b44ca218f6e0323e05202a1fed7 (Merge pull request #53 from ahaselsteiner/issue#27-unique-email). This merge was done with a pull request by @topape and reviewed by me. Since latexify worked in the master's earlier version d2e969c806af6bbce17419ff29ff075719a50a6d (add "build passing" badge to README.md), I think this is where the problem started.

After we solved this I think we should introduce some tests, which cover the GUI. If Travis finds something like this, we save a lot of time.
